common/common_test.go

```diff
@@ -26,3 +28,43 @@ func TestValidateImageName(t *testing.T) {
 		})
 	}
 }
+
+func Test_proxyArgs(t *testing.T) {
```

Comment:
Nit: There are a bunch of tools (eg. golint) which will complain about underscores in names. Might be nice to avoid.

Reply:
Scratch that... looks like it doesn't apply that rule to test functions.